src/main/java/seedu/address/logic/parser/DeleteMeetingCommandParser.java

```diff
@@ -44,6 +44,14 @@ public DeleteMeetingCommand parse(String args) throws ParseException {
             throw new ParseException(DeleteMeetingCommand.MESSAGE_INVALID_BLANK_PERSON_INDEX);
         }
 
+        // Checks if meeting Index is blank
+        if (meetingIndexString.isEmpty()) {
+            throw new ParseException(DeleteMeetingCommand.MESSAGE_INVALID_BLANK_MEETING_INDEX);
+        }
```

Comment:
```suggestion
        if (meetingIndexString.isBlank()) {
            throw new ParseException(DeleteMeetingCommand.MESSAGE_INVALID_BLANK_MEETING_INDEX);
        }
```
Perhaps this should be `meetingIndexString.isBlank()` instead of `.isEmpty()`? `isEmpty()` would return true if `meetingIndexString` has a length of 0 which means that if `meetingIndexString` was just white space, no exception would be thrown. Alternatively, you could do `meetingIndexString.trim().isEmpty()` which would remove the white spaces if applicable.